packages/react-component-library/package.json

```diff
@@ -32,8 +31,8 @@
     "prepare": "yarn build",
     "storybook:static": "build-storybook -c .storybook -o .static_storybook",
     "storybook": "start-storybook -p 6006",
-    "test:watch": "NODE_ENV=test jest --watch",
```

Comment:
Why have you removed the `NODE_ENV=test` setting this environment variable ensures that the correct transforms are applied as specified within the .babelrc file:

https://github.com/Royal-Navy/standards-toolkit/blob/develop/packages/react-component-library/.babelrc

```json
"env": {
    "test": {
      "plugins": [
        "@babel/plugin-transform-modules-commonjs"
      ]
    }
  }
```

Without this the tests will fail because Node cannot interprety ESM modules it needs them to be transformed to CommonJS.

Reply:
They still pass, though this may be because the babel transform is in the jest config.

I wonder if there is another way to do this without the NODE_ENV bit in the command line as Powershell and CMD don't support that syntax.. though I'll see if there is a way